package.json

```diff
@@ -1,11 +1,9 @@
 {
-  "main": "node_modules/expo/AppEntry.js",
   "scripts": {
-    "start": "expo start",
-    "android": "expo start --android",
-    "ios": "expo start --ios",
+    "start": "react-native start",
```

Comment:
😮  it's not expo no more. the Bare workflow is basically using the react-native cli